htdocs/index.php

```diff
@@ -29,7 +29,8 @@
 // Middleware that happens on every request. This doesn't include
 // any authentication middleware, because that's done dynamically
 // based on the module router, depending on if the module is public.
-$middlewarechain = (new \LORIS\Middleware\ContentLength())
+$middlewarechain = (new \LORIS\Middleware\ExceptionHandlingMiddleware())
+    ->withMiddleware(new \LORIS\Middleware\ContentLength())
```

Comment:
Are you sure these are the right order? I can't remember if the chain is a stack or a queue, but can you make sure this calculates the content-length of the error page, and not of the pre-modified data?

Reply:
> 1.4 Handling Exceptions
It is RECOMMENDED that any application using middleware includes a component that catches exceptions and converts them into responses. **This middleware SHOULD be the first component executed and wrap all further processing to ensure that a response is always generated.**

https://www.php-fig.org/psr/psr-15/

If that's any consolation

Reply:
Inverting the order doesn't seem to change the Content-Length value for the 500 response error page (see screenshot in OP). I made sure to refresh with Cmd + Shift + R to clear the cache

Reply:
I don't see the network dev console/http response headers in the screenshot, how can I check the Content-Length value?

Reply:
I just meant that was the page I was testing, not that the content-length is there.

They match in manual testing for me. Maybe you can confirm locally to verify?